flow/docs/grpc-api.md

```
@@ -0,0 +1,2396 @@
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| id | [UUID](#v1-UUID) |  |  |
+| name | [string](#string) |  |  |
+| description | [string](#string) |  |  |
+| operation_type | [OperationType](#v1-OperationType) |  |  |
+| operation_code | [string](#string) |  | Specific operation code (e.g., &#34;power_on&#34;, &#34;upgrade&#34;) |
+| rule_definition_json | [string](#string) |  | JSON-encoded RuleDefinition |
+| is_default | [bool](#bool) |  |  |
+| created_at | [google.protobuf.Timestamp](#google-protobuf-Timestamp) |  |  |
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)